File: trail1/총합 구하기/find-the-total.cs
using System;

public class Codetree
{  
    public static void Main()
    {
        string[] arr = Console.ReadLine().Split();
        var(A, B) = (int.Parse(arr[0]), int.Parse(arr[1]));
        int ssum = 0;
        for (int i=A; i<=B; i++) {if (i % 6 == 0 && i % 8 != 0) {ssum += i;}}
        Console.WriteLine(ssum);
    }
}

File: trail1/총합 구하기/find-the-total.cpp
#include <iostream>
using namespace std;

int main() {
    int A, B;
    cin >> A >> B;
    int ssum = 0;
    for (int i=A; i<=B; i++) {
        if (i % 6 == 0 && i % 8 != 0) {ssum += i;}
    }
    cout << ssum << " ";
    return 0;
}